data/helpers.d/utils

```diff
@@ -319,8 +324,17 @@ ynh_add_config () {
 
     ynh_backup_if_checksum_is_different --file="$destination"
 
+    # Make sure to set the permissions before we copy the file
+    # This is to cover a case where an attacker could have
+    # created a file beforehand to have control over it
+    # (cp won't overwrite ownership / modes by default...)
+    touch $destination
+    chown root:root $destination
+    chmod 750 $destination
```

Comment:
```suggestion
    chmod -R 440 $destination
```

Reply:
Will set it to `640` without -R because that's a file and not a directory